Sources/PrimerSDK/Classes/Data Models/PrimerSettings.swift

```diff
@@ -156,8 +156,21 @@ public class PrimerApplePayOptions: Codable {
                 merchantName: String?,
```

Comment:
Public initialiser does not include `shippingOptions`, and `ShippingOptions` is internal. Internals will work as though merchant did not set these options, and merchant cannot set them.